packages/node_modules/@node-red/editor-client/src/js/nodes.js

```diff
@@ -1032,23 +1032,31 @@ RED.nodes = (function() {
         return {nodes:removedNodes,links:removedLinks, groups: removedGroups, junctions: removedJunctions};
     }
 
+    /**
+      * Add a Subflow to the Workspace
+      *
+      * @param {object} sf The Subflow to add.
+      * @param {boolean|undefined} createNewIds Whether to update the name.
+      */
     function addSubflow(sf, createNewIds) {
```

Comment:
Is `createNewIds` supposed to only update the name or also the id (inside this function)?
Note: this function is exported.

Reply:
It is probably just a lazily named argument because it *is* `createNewIds` from the calling function. The ids will have already been regenerated by the time this is called - on the basis that it is a duplicate subflow we're importing a copy. As a copy, we want to update the name to make sure its unique.

Reply:
Ok thanks, maybe we can call it something else?